pom.xml

```diff
@@ -250,7 +254,7 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>3.5.4</version>
+        <version>4.0.0</version>
```

Comment:
Inconsistent jackson versions 2.9.6 and 2.11.2:
```
mvn dependency:tree -Dincludes=*jackson*

[INFO] org.folio:edge-common:jar:3.1.0-SNAPSHOT
[INFO] +- io.vertx:vertx-core:jar:4.0.0:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.11.3:compile
[INFO] +- com.fasterxml.jackson.core:jackson-annotations:jar:2.11.3:compile
[INFO] +- com.fasterxml.jackson.dataformat:jackson-dataformat-xml:jar:2.9.6:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.11.3:compile
[INFO] |  \- com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.9.6:compile
[INFO] \- com.amazonaws:aws-java-sdk-ssm:jar:1.11.313:compile
[INFO]    \- com.amazonaws:aws-java-sdk-core:jar:1.11.313:compile
[INFO]       \- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.11.3:compile
```

```suggestion
        <version>4.0.0</version>
        <!-- Update jackson-bom accordingly: https://github.com/vert-x3/vertx-dependencies/blob/master/pom.xml -->
```

Reply:
Good catch. I'll update junit as well.

Reply:
You fixed it, thanks, can be closed.